testing/integration-test.js

```diff
@@ -0,0 +1,30 @@
+const getPortSync = require('get-port-sync')
+
+exports.register = function ({ test, context, startupTasks, server }) {
+  let httpServerPort
+
+  try {
+    console.log('Attempting to obtain a random port to start the test HTTP server on...')
+    httpServerPort = getPortSync()
```

Comment:
Just so you know, when you start an http server without specifying a port, a random port is automatically assigned. Nbd tho

Reply:
Just in case any other reviewers are curious, this is necessary because when we load the service config, we have port `8080` set as the default. Without this, every test server will try to start on port `8080`.